composer.json

```diff
@@ -32,7 +32,7 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.19 || ^3.8",
         "mf2/mf2": "^0.5.0",
-        "phpstan/phpstan": "^1.11",
+        "phpstan/phpstan": "~1.12.2",
```

Comment:
We will want to update it regularly but I guess we can do that later. Not sure how well [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy) works with tilde constraint, though.